content/dev/dev-guide.md

```diff
@@ -0,0 +1,264 @@
+- With that saved we now have everythign we need in the `hello` directory.
+##### Installing The New Desk Into Fake Zod
+- First we're going to create a new fake zod that's bootstrapped with a pill from our urbit-git repo. The pills are stored in git lfs and need to be pulled into our repo first.
+  - ```bash
+    cd ~/urbit-git
+    git lfs install
+    git lfs pull
+    ```
+- Next we're going to create the fake zod using the `multi-brass.pill` we just pulled down.
+  - ```bash
+    cd ~/urbit-dev
+    ./urbit -F zod -B ~/urbit-git/bin/multi-brass.pill
+    ```
+- Once our new fake zod is booted we'll create and mount a new `%hello` desk in dojo.
+  - ```bash
+    |merge %hello our %base
+    |mount %hello
+    ```
+- We should now see `hello` inside our `zod` directory.
+- Currently it's just a clone of `%base` so let's delete its contents.
+  - ```bash
+    cd ~/urbit-dev/zod
+    rm -r hello/*
+    ```
+- Next we'll copy the contents of the `hello` desk we created earlier. We'll need to use `cp -LR` to resolve all the symlinks.
+  - ```bash
+    cp -LR ~/urbit-git/pkg/hello/* hello/
+    ```
+- Back in the `zod` dojo we can commit the changes and install the desk.
+  - ```bash
+    |commit %hello
+    |install our %hello
+    ```
+  - You'll see `docket: awaiting manual glob for %hello desk` this is because our `desk.docket-0` files includes a `%glob-ames` clause which specifies our ship as the source, so it's waiting for us to upload the glob.
+- Open a browser and navigate to `http://localhost:80` and login with the default fake zod code `lidlut-tabwed-pillex-ridrup`. You'll see our tile, but it says installing with a spinner due to the missing glob.
+##### Create Files for Glob
+We'll now create the files for the glob. We'll use a very simple static HTML page that just displayes "Hello World!" and an image. Typically we'd have a more complex JS web app that talked to apps on our ship through Eyre's channel system, but for the sake of simplicity we'll forgo that. Let's hop back in the Unix terminal.
+ - ```bash
+   cd ~
+   mkdir hello-glob
+   cd hello-glob
+   mkdir img
+   wget -P img https://media.urbit.org/docs/userspace/dist/pot.svg
+   ```
+- We've grabbed an image to use in our "Hello world!" page. The next thing we need to add is an `index.html` file in the root of the folder. The `index.html` file is mandatory; it's what will be loaded when the app's tile is clicked. Let's open our preferred editor and create it.
+  - ```bash
+    nano index.html
+    ```
+- Paste the following into the editor and save it.
+  - ```html
+    <!DOCTYPE html> <html> <head> <style> div { text-align: center; }
+    </style> </head> <title>Hello World</title> <body> <div>
+    <h1>Hello World!</h1> <img src="img/pot.svg" alt="pot" width="219"
+    height="196" /> </div> </body> </html>
+    ```
```

Comment:
The `<title>` probably shouldn't live outside the `<head>`. Everything else here is just a stylistic change, but should help make this slightly less opaque.

```suggestion
  - ```html
    <!DOCTYPE html>
    <html>
      <head>
        <title>Hello World</title>
        <style>body { text-align: center; }</style>
      </head>
      <body>
        <h1>Hello World!</h1>
        <img src="img/pot.svg" alt="pot" width="219" height="196" />
      </body>
    </html>
    ```
```

Reply:
Also I feel like we shouldn't lose the nuance of docket's glob capability mostly being a loom limits workaround. Questions like "why not copy this in through clay" will inevitably arise, and for this specific example that's totally reasonable (no big js blobs, after all).

Reply:
This nuance was not clear to me from the existing guide - it sounds like a good note to add, but I need to ask you more about it in order to understand it well enough to write it up clearly.

Reply:
Added context based on 1:1 with Mark